assets/js/admin/course-pricing-promo-sidebar.js

```diff
@@ -0,0 +1,66 @@
+import { __ } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { escapeHTML } from '@wordpress/escape-html';
+import { Button, ExternalLink } from '@wordpress/components';
+
+/**
+ * Course Pricing Promo Sidebar component.
+ */
+
+const CoursePricingPromoSidebar = () => {
+	/**
+	 * Filters to get description for pricing component.
+	 *
+	 * @since 4.1.0
+	 *
+	 * @hook  senseiCoursePricingDescription This hook allows to pass a string value for the course pricing promo description.
+	 * @return {string} 					 Description text for course pricing promo sidebar.
+	 */
+	const description = applyFilters(
+		'senseiCoursePricingDescription',
+		__(
+			'Sell this course using WooCommerce - integrates with subscriptions, memberships, affiliates, and more.'
```

Comment:
The second parameter (`sensei-lms`) is missing here.

Reply:
uh, [here](https://github.com/Automattic/sensei/pull/4810/commits/2d8fa3640baf4711ad7a01cdb72cac37fd13c7cc)